decidim-core/app/mailers/decidim/newsletter_mailer.rb

```diff
@@ -11,7 +11,7 @@ class NewsletterMailer < ApplicationMailer
 
     helper_method :cell
 
-    def newsletter(user, newsletter, preview: false)
+    def newsletter(user, newsletter, preview = nil)
```

Comment:
This needs to be written like this as, if we keep it as a keyword argument, ruby and rails will throw an error like "Wrong number of arguments 3 for 2". 
If we change it to `preview = false`, rubocop would complain that needs to be "preview: false", which is leading to the initial issue. 
I would not want to create an Rubocop/disable for this line.